pyout.py

```diff
@@ -592,6 +609,18 @@ def _setup_fields(self):
 
             self._fields[column] = field
 
+    @property
+    def ids(self):
+        if self._ids is None:
+            if self._columns:
+                return [self._columns[0]]
```

Comment:
from reading this difference alone not clear why a single first element placed into a list is equivalent to `ids` and what `ids` then supposed to be. I guess a comment and a docstring for "ids" method/property could be of great help

Reply:
ok

Reply:
I ended up going with just the docstring since adding a comment seemed
redundant at that point.